docs/flowcollector-flows-netobserv-io-v1beta1.adoc

```diff
@@ -845,7 +900,7 @@ Type::
 
 | `authToken`
 | `string`
-| AuthToken describe the way to get a token to authenticate to Loki. DISABLED will not send any token with the request. HOST will use the local pod service account to authenticate to Loki. FORWARD will forward user token, in this mode, pod that are not receiving user request like the processor will use the local pod service account. Similar to HOST mode. When using the Loki Operator, set it to `HOST` or `FORWARD`.
+| AuthToken describe the way to get a token to authenticate to Loki. DISABLED will not send any token with the request. HOST - _deprecated (*)_ - will use the local pod service account to authenticate to Loki. FORWARD will forward the user token for authorization. When using the Loki Operator, this should be set to `FORWARD`.
```

Comment:
Really minor thing, but IBM Style Guide says to use "must" instead of "should", so is it OK to change this one small thing in the docs version of this 😬 ?

Reply:
@skrthomas I'll send a new PR with this + a couple of other changes